cosmos/pocket-testnet.json

```
@@ -0,0 +1,49 @@
+{
+
+  "chainId": "pocket-beta",
+  "chainName": "Pocket Network",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.jpg",
```

```suggestion
  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket-testnet/chain.png",
```
jpg -> png

Also, the directory where the `chain.png` file is located should be renamed from `pocket-mainnet` to `pocket`.